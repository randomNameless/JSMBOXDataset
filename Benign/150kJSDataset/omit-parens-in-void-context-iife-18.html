<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>omit-parens-in-void-context-iife-18.html</title>
</head>
<body>

<script>
/*{
    "options": {
        "preserveCompletionValue": true
    }
}*/
do {
  // do not optimize it
  (function () {
    print('ok');
  }());
} while (cond);

</script>

</body>
</html>
