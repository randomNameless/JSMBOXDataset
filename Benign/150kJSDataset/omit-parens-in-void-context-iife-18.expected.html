<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>omit-parens-in-void-context-iife-18.expected.html</title>
</head>
<body>

<script>
do(function(){print('ok')}());while(cond)

</script>

</body>
</html>
